tests/utils/compute_groundtruth.cpp

```diff
@@ -421,24 +427,104 @@ inline void parse_label_file_into_vec(size_t &line_cnt, const std::string &map_f
 }
 
 template <typename T>
-int aux_main(const std::string &base_file, const std::string &label_file, const std::string &query_file,
-             const std::string &gt_file, size_t k, const std::string &filter_label, const std::string &universal_label,
-             const diskann::Metric &metric, const std::string &tags_file = std::string(""))
+std::vector<std::vector<std::pair<uint32_t, float>>> processUnfilteredParts(const std::string &base_file,
```

Comment:
Can we have two compute GT tools? One for unfiltered and filtered?

Reply:
Now we have two GT files 
For Unfiltered : compute_groundtruth.cpp
for Filtered : compute_groundtruth_for_filtered.cpp